src/Symfony/Component/Yaml/Resources/bin/yaml-lint

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Runs the Yaml lint command.
+ *
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Yaml\Command\LintCommand;
+
+function includeIfExists(string $file): bool
+{
+    return file_exists($file) && include $file;
+}
+
+if (
+    !includeIfExists(__DIR__ . '/../../../../autoload.php') &&
```

Comment:
Which use case is this line supposed to cover?

Reply:
installing `symfony/symfony` as a vendor

Reply:
Ah no, this is `smfony/yaml` as vendor. But, don't we need the same with eight `..` segments for `symfony/symfony`?

Reply:
we do

Reply:
No we don't need eight `..` segments, because `symfony/symfony` is the root dir in this case including the `src` folder with a `Symfony/Component/Yaml/Resource/bin` and the `vendor/autoload.php` which leads to 6 `..` as seen in line 34.

Reply:
@jschaedl we are talking about `symfony/symfony` being installed as a dependency.

Reply:
@jschaedl Is this resolved actually?

Reply:
@chalasr I discussed this with @xabbuh and we both agreed that these checks are correct.  The same code is actually used here: https://github.com/symfony/symfony/blob/e60a876201b5b306d0c81a24d9a3db997192079c/src/Symfony/Component/VarDumper/Resources/bin/var-dump-server#L34 as well, so yes this should be resolved.